CMakeLists.txt

```diff
@@ -59,6 +65,11 @@ add_library(mediaGraph
                           timestamp
                           thread_primitives
                          )
+    if (ENABLE_EASY_PROFILER)
```

Comment:
I'll change it but In this case I think they both do the same.

Reply:
not the same. You might invoke cmake with ENABLE_EASY_PROFILER=ON, but then easy profiler is not found.

Reply:
that's why I added a `find_package(easy_profiler REQUIRED)` so that doesn't happen.
I see your logic, it makes sense and it may be more robust, though it depends what we want.